websocket-native/src/main/java/org/ballerinalang/net/websocket/WebSocketConstants.java

```diff
@@ -145,26 +139,24 @@
     public static final String PARAM_TYPE_STRING = "string";
 
     // Strand meta data
-    public static final StrandMetadata ON_OPEN_METADATA =
-            new StrandMetadata(BALLERINA_ORG, PACKAGE_WEBSOCKET, WEBSOCKET_MODULE_VERSION, RESOURCE_NAME_ON_OPEN);
-    public static final StrandMetadata ON_TEXT_METADATA =
-            new StrandMetadata(BALLERINA_ORG, PACKAGE_WEBSOCKET, WEBSOCKET_MODULE_VERSION,
-                    RESOURCE_NAME_ON_TEXT_MESSAGE);
-    public static final StrandMetadata ON_BINARY_METADATA =
-            new StrandMetadata(BALLERINA_ORG, PACKAGE_WEBSOCKET, WEBSOCKET_MODULE_VERSION,
-                    RESOURCE_NAME_ON_BINARY_MESSAGE);
-    public static final StrandMetadata ON_PING_METADATA =
-            new StrandMetadata(BALLERINA_ORG, PACKAGE_WEBSOCKET, WEBSOCKET_MODULE_VERSION, RESOURCE_NAME_ON_PING);
-    public static final StrandMetadata ON_PONG_METADATA =
-            new StrandMetadata(BALLERINA_ORG, PACKAGE_WEBSOCKET, WEBSOCKET_MODULE_VERSION, RESOURCE_NAME_ON_PONG);
-    public static final StrandMetadata ON_CLOSE_METADATA =
-            new StrandMetadata(BALLERINA_ORG, PACKAGE_WEBSOCKET, WEBSOCKET_MODULE_VERSION, RESOURCE_NAME_ON_CLOSE);
-    public static final StrandMetadata ON_ERROR_METADATA =
-            new StrandMetadata(BALLERINA_ORG, PACKAGE_WEBSOCKET, WEBSOCKET_MODULE_VERSION, RESOURCE_NAME_ON_ERROR);
+    public static final StrandMetadata ON_OPEN_METADATA = new StrandMetadata(BALLERINA_ORG, PACKAGE_WEBSOCKET,
```

Comment:
We may have to create these metadata inside `setModule()` of moduleInit to avoid these being created for every call. Also at this point `ModuleUtils.getWebsocketModule()` is null IINM

Reply:
Fixed with d2a513ddefd04a9df586a929b60cee5d1288fc9f